Products/ZenModel/FileSystem.py

```diff
@@ -110,11 +110,23 @@ class FileSystem(OSComponent):
         )
 
 
+    def getNoneRootTotalBlocks(self):
```

Comment:
It's not clear what a `noneRootTotalBlocks` property is.  Should this be a separate method?